unittest/python/test_rk4_comp.py

```diff
@@ -0,0 +1,168 @@
+import numpy as np
+
+import pinocchio
+import example_robot_data
+
+import crocoddyl
+import crocoddyl_legacy
```

Comment:
You will need to include the utils module, see this example: https://github.com/loco-3d/crocoddyl/blob/master/unittest/bindings/test_actions.py#L10

Reply:
done